template/src/app.ts

```diff
@@ -1,10 +1,10 @@
-import { registerHtml } from "tram-one";
+import { registerHtml, TramOneComponent } from "tram-one";
 import logo from "@tram-one/tram-logo/dist/element";
 import "./app.css";
 
 const html = registerHtml();
 
-export default () => {
+const app: TramOneComponent = () => {
```

Comment:
This is honestly the most significant part of the change, we now pull in one of the types included in Tram-One, and type our component with it.